### DISCUSSION TITLE
Editir Element returning null

### DISCUSSION THREAD

**Discussion by PiotrAdamowicz** [vueform/vueform#281] [Questions] [answered]
I have a problem with getting input data from the Editor element.
When I submit using @submit vue event the best I get is `{ rte: null }`

My component:
```
<script setup>
  import { ref } from 'vue';

  const schema = ref({
    rte: {
      type: 'editor',
      placeholder: 'Enter your message...',
    },
    button: {
      'button-label': 'Submit',
      type: 'button',
      'button-class': 'font-semibold',
      submits: true,
    },
  });
  function onSubmit(form$) {
    console.log(form$.empty);
    console.log('Form Data:', form$.data);
    console.log('Editor Data:', form$.data.rte);
    form$.submitting = false; // Sets button as not loading
  }
</script>
<template>
  <div class="">List of messages</div>
  <div class="">
    <Vueform @submit="onSubmit" :schema="schema" :endpoint="false"></Vueform>
  </div>
</template>

```
Console output I get:

![image](https://github.com/user-attachments/assets/3f6fe2d5-b644-4f7f-b4ca-d224ed428322)

Input looks like this:
![image](https://github.com/user-attachments/assets/41105142-02ee-437a-814b-d8ee5c561937)

I started some testing on this and got some results when changing theme from Bootstrap to Tailwind, sadly my main project setup is on Bootstrap so I can't use that.

Also config file:
```
import en from '@vueform/vueform/locales/en';
import bootstrap from '@vueform/vueform/dist/bootstrap';
import { defineConfig } from '@vueform/vueform';

export default defineConfig({
  displayErrors: false,
  theme: bootstrap,
  locales: { en },
  locale: 'en',
  presets: {
    default: {
      addClasses: {
        elementLayout: 'mb-5',
      },
    },
  },
  usePresets: ['default'],
});

```

**Comment by adamberecz** [comment, marked as answer]
Which version are you using? This got fixed recently.

**Reply by PiotrAdamowicz** [reply]
"@vueform/vueform": "^1.9.10",

**Reply by adamberecz** [reply]
It got fixed in [`1.9.11`](https://github.com/vueform/vueform/releases/tag/1.9.11)

**Reply by PiotrAdamowicz** [reply]
Thanks for the quick answer, just upgraded and it works fine.
